src/api/searchApi.ts

```diff
@@ -195,6 +196,7 @@ export interface FetchResultsParams {
   [ResultParam.Course]?: string | null;
   [ResultParam.CristinIdentifier]?: string | null;
   [ResultParam.Doi]?: string | null;
+  [ResultParam.ExcludeSubunits]?: 'true' | null;
```

Comment:
Naturlig at dette er `boolean`, på samme måte som det er løst for ticket search?

```suggestion
  [ResultParam.ExcludeSubunits]?: boolean | null;
```

Reply:
Da kan det settes som noe sånt for `FetchResultsParams`: `excludeSubunits: Boolean(params.get(ResultParam.ExcludeSubunits))`